bluesky/plans.py

```diff
@@ -1153,6 +1166,75 @@ def finalize_wrapper(plan, final_plan, *, pause_for_debug=False):
     return ret
 
 
+def teef_wrapper(plan, *,
```

Comment:
This name is so bad it's good.

Reply:
Oh actually — I wasn’t going to suggest changing this but I have a good one: contingency_plan!